### ISSUE TITLE
$('a').getText() returns empty string

### ISSUE THREAD

**Issue by diachedelic** [webdriverio/webdriverio#4253]
**Environment (please complete the following information):**
 - **WebdriverIO version: 5.11.9
 - **Mode: WDIO testrunner
 - **If WDIO Testrunner, running sync/async: sync
 - **Node.js version: v11.15.0
 - **NPM version: v6.7.0
 - **Platform name and version: MacOS 10.14.5

**Config of WebdriverIO**
See repro

**Describe the bug**
Calling `.getText()` on an anchor element returns an empty string.

**To Reproduce**

```
git clone git@github.com:diachedelic/wdio-repros.git &&
  cd wdio-repros &&
  git checkout anchor-get-text &&
  yarn &&
  yarn test
```

**Expected behavior**
getText returns the text within

**Log**
Run repro

**Comment by mgrybyk** [comment]
@diachedelic I think you are doing something wrong here

First of all you are not using getText but getHTML that should return an empty string in your case.
Secondly you are using really old versions .

https://github.com/diachedelic/wdio-repros/blob/anchor-get-text/test/anchorGetText.js

Am I missing something?

**Comment by christian-bromann** [comment]
I also can't reproduce this:

```
wdio repl chrome
2019-07-25T07:44:06.536Z INFO webdriver: [POST] http://localhost:4444/wd/hub/session
2019-07-25T07:44:06.538Z INFO webdriver: DATA { capabilities: { alwaysMatch: { browserName: 'chrome' }, firstMatch: [ {} ] },
  desiredCapabilities: { browserName: 'chrome' } }

The execution has stopped!
You can now go into the browser or use the command line as REPL
(To exit, press ^C again or type .exit)

› browser.url('https://webdriver.io/docs/api/browser/url.html')
2019-07-25T07:44:11.282Z INFO webdriver: COMMAND navigateTo("https://webdriver.io/docs/api/browser/url.html")
2019-07-25T07:44:11.283Z INFO webdriver: [POST] http://localhost:4444/wd/hub/session/42fa796bf13974d807153aaec53fac13/url
2019-07-25T07:44:11.283Z INFO webdriver: DATA { url: 'https://webdriver.io/docs/api/browser/url.html' }
null
› const anchor = browser.$('a.navItem')
2019-07-25T07:44:16.771Z INFO webdriver: COMMAND findElement("css selector", "a.navItem")
2019-07-25T07:44:16.772Z INFO webdriver: [POST] http://localhost:4444/wd/hub/session/42fa796bf13974d807153aaec53fac13/element
2019-07-25T07:44:16.772Z INFO webdriver: DATA { using: 'css selector', value: 'a.navItem' }
2019-07-25T07:44:16.795Z INFO webdriver: RESULT { ELEMENT: '0.952662039509093-1' }
undefined
› anchor.getHTML()
2019-07-25T07:44:24.498Z INFO webdriver: COMMAND executeScript("return (function getHTML(element, includeSelectorTag) {
  return element[includeSelectorTag ? 'outerHTML' : 'innerHTML'];
}).apply(null, arguments)", <object>)
2019-07-25T07:44:24.499Z INFO webdriver: [POST] http://localhost:4444/wd/hub/session/42fa796bf13974d807153aaec53fac13/execute
2019-07-25T07:44:24.499Z INFO webdriver: DATA { script: 'return (function getHTML(element, includeSelectorTag) {\n  return element[includeSelectorTag ? \'outerHTML\' : \'innerHTML\'];\n}).apply(null, arguments)',
  args:
   [ { 'element-6066-11e4-a52e-4f735466cecf': '0.952662039509093-1',
       ELEMENT: '0.952662039509093-1' },
     true ] }
2019-07-25T07:44:24.505Z INFO webdriver: RESULT <a class="navItem" href="/docs/api.html">API Docs</a>
'<a class="navItem" href="/docs/api.html">API Docs</a>'
› anchor.getText()
2019-07-25T07:44:31.283Z INFO webdriver: COMMAND getElementText("0.952662039509093-1")
2019-07-25T07:44:31.283Z INFO webdriver: [GET] http://localhost:4444/wd/hub/session/42fa796bf13974d807153aaec53fac13/element/0.952662039509093-1/text
2019-07-25T07:44:31.324Z INFO webdriver: RESULT API Docs
'API Docs'
```

**Comment by diachedelic** [comment]
Here is my log (I know the package.json had old versions listed but if you check the yarn.lock it is webdriverio@5.11.7):

https://gist.github.com/diachedelic/6733a5df39859cb04a4366d8a7e92aff

@christian-bromann I've clarified the test case and have swapped from yarn to npm, please try running this and tell me what you get:

```
git clone git@github.com:diachedelic/wdio-repros.git &&
  cd wdio-repros &&
  git checkout anchor-get-text &&
  npm i &&
  npm test
```
It certainly fails for me - I've checked in `package-lock.json` for your perusal.

**Comment by christian-bromann** [comment]
@diachedelic can you try `getHTML`? Not why this is but it is definitely not a WebdriverIO bug since all requests are made properly.

**Comment by diachedelic** [comment]
@christian-bromann getHTML works fine - it returns the full HTML of the anchor, including it's text (that's why I included it in my previous revision - to show that there definitely was text in there)

**Comment by christian-bromann** [comment]
@diachedelic then this is a bug in the driver

**Comment by diachedelic** [comment]
@christian-bromann ok, thanks. For reference, I was using Chrome `76.0.3809.80 (Official Build) beta (64-bit)` so it was probably a driver mismatch with chromedriver which only supports Chrome 75

**Comment by laclance** [comment]
How did you fix it please? I have same issue with v84.0.1. I copied https://webdriver.io/docs/gettingstarted.html and this is what i get: 
```
const el = browser.$('label.text-2l');
expect(el.getHTML()).toEqual('Sign In');
expect(el.getText()).toEqual('Sign In');
```
```
Expected: "Sign In"
Received: "<label class=\"text-2l\">Sign In</label>"
```
Expected: "Sign In"
Received: ""
